package.json

```diff
@@ -32,5 +32,9 @@
     "jshint-stylish": "^0.4.0",
     "run-sequence": "~0.3.6",
     "yargs": "^1.3.3"
+  },
+  "dependencies": {
+    "moment": "^2.10.6",
```

Comment:
It is required as soon as you want to reformat the date in the Pikaday datepicker.